Sources/SwiftCliCore/ChartConstructor.swift

```diff
@@ -59,6 +73,32 @@ public class ChartConstructor {
         middle_row_bars += "\n"
         return(middle_row_bars)
     }
+    
+    public func make_middle_row_stitch_count_change(width: Int, right_difference: Int, left_difference: Int) -> String {
+        
+        
+        var middle_row_bars = ""
+        if (width > 0){
+            middle_row_bars += "├"
+            if (right_difference < 0){
+            if (width > 1){
+                for _ in 1...(width) {
+                    middle_row_bars += "─┼"
```

Comment:
Instead of doing all these nested `for` loops to repeat characters, it may be easier to use this parameter in the `String.init` method: https://developer.apple.com/documentation/swift/string/2427723-init

Reply:
This is great! I implemented it in all my auxiliary methods for assembling the chart string in ChartConstructor.